src/common/models/Projection.ts

```diff
@@ -310,6 +311,9 @@ export class Projection {
 
     this.currentHsaIcuInfo = summaryWithTimeseries.actuals.hsaIcuBeds;
     this.currentHsaHospitalInfo = summaryWithTimeseries.actuals.hsaHospitalBeds;
+    // For counties, in most cases we will want to access hsaHospitalBeds (usually via currentHsaHospitalInfo)
```

Comment:
Normally comments explain the code following it, but this is more of a sidenote warning people they may not want to use this (I think?).  Perhaps rephrase:

```
// NOTE: For counties you likely want to use hsaHospitalBeds (e.g. via currentHsaHospitalInfo) instead of this.
```

Though still, it seems a bit unlikely that somebody will see this comment at the right time to help them prevent a mistake.  You could move it to a jsdoc comment on the property declaration, e.g.:

```
/** Gets the latest weekly covid admissions data for this location. Note that for counties you likely want to use the HSA data (e.g. via currentHsaHospitalInfo) instead of this. */
readonly currentWeeklyCovidAdmissions: number | null;
```

Reply:
Ah, cool, thanks for the distinction. Updated to use a jsdoc comment